lib/e164/lookuptable.rb

```diff
@@ -1,10 +1,14 @@
-# This file is automatically generated by `rake rebuild`
+# USED TO BE -> This file is automatically generated by `rake rebuild`, library is no longer maintained
+# this file is manually edited by pulling down the NAPA database and editing this:
+# https://nationalnanpa.com/reports/area_code_relief_planning.html
+
+# this only updated the US country codes.  Everything else is < 2015.  We generally only care if it's a USA phone number
```

Comment:
Whoa... We absolutely care about canada codes.

Reply:
We don't in api.  The code is only checking for the country to match US.  So if we return nil or something else, it will fail the device validation.

```ruby
  def supported_country?(country)
    supported_countries.include?(country || 'US')
  end

  # Public: Returns true if the numeric country code is supported for SMS.
  #
  # code - The numeric (phone) country code.
  #
  # Returns Boolean.
  def supported_sms_region?(number)
    byebug

    # E164 is fundamentally flawd because it does a parse for country codes from the BACK of the string.
    # this means for a perfectly valid area code of 689 we get :
    # E164.lookup('6892109310')
    # ["PF", "French Polynesia"]
    # (byebug) E164.lookup('16892109310')
    # nil
    country_code, _ = E164.lookup(number.gsub(/\D/, '')) # E164 doesn't want formatting cruft
    country_code && (supported_country? country_code)
  end```